index.js

```diff
@@ -13,9 +16,9 @@ const _SERVER_PORT = process.env.PORT || 3000;
 app.use(bodyParser.json());
 app.use(cors());
 // Add API routes
-app.use('/api/auth', require('./server/api/auth.js'));
```

Comment:
I think this will really help in terms of code sanctity 🥇

Reply:
Yeah probably better to have a variable then inline usage.